tensorflow/compiler/tests/binary_ops_test.py

```diff
@@ -34,7 +34,6 @@
 from tensorflow.python.ops import math_ops
 from tensorflow.python.ops import nn_ops
 from tensorflow.python.platform import googletest
-from tensorflow.python.platform import test as test_lib  # pylint: disable=unused-import
```

Comment:
This is not needed in our repo but it is needed and present in the upstream repo.  This import is used in the upstream repo because of an apparent lack of support for gemm operations in rocm.  Our repo has gemm operations but apparently the upsteam repo has not been updated.